sdk/spring/azure-spring-boot-starter/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 3.2.0-beta.1 (Unreleased)
+## 3.1.0 (2021-01-20)
```

Comment:
We need to add all spring boot related changelogs here.

Reply:
Updated.